app/views/collections/say-brother.md

```diff
@@ -22,3 +22,11 @@ conversations with distinct voices from the African American community.
 ## Records
 
 [](http://localhost:3000/catalog?f[special_collection_tags][]=sb_record)
+
+## Related Content
+
+[Basic Black](http://wgbh.org/basicblack)
```

Comment:
@mccalluc: Can you add www. before wgbh.org? That will fix it.


________________________________
From: mccalluc <notifications@github.com>
Sent: Wednesday, March 9, 2016 11:38 AM
To: WGBH/openvault3
Cc: Casey Davis
Subject: Re: [openvault3] Add-related-content-to-say-brother (#400)


In app/views/collections/say-brother.md<https://github.com/WGBH/openvault3/pull/400#discussion_r55546953>:

> @@ -22,3 +22,11 @@ conversations with distinct voices from the African American community.
>  ## Records
>
>  [](http://localhost:3000/catalog?f[special_collection_tags][]=sb_record)
> +
> +## Related Content
> +
> +[Basic Black](http://wgbh.org/basicblack)


@caseyedavis12<https://github.com/caseyedavis12> : This redirects to plain wgbh.org. Do you have a replacement?

-
Reply to this email directly or view it on GitHub<https://github.com/WGBH/openvault3/pull/400/files#r55546953>.